eo2js/src/commands/link.js

```diff
@@ -52,22 +52,33 @@ const pckg = function(options) {
 const link = function(options) {
   options = {...program.opts(), ...options}
   const project = path.resolve(options.target, options.project)
+  console.log(`Building npm project in: ${project}`)
+
+  console.log('Creating package.json...')
   fs.writeFileSync(
     path.resolve(project, 'package.json'),
     JSON.stringify(pckg(options))
   )
+
+  console.log('Installing dependencies...')
   execSync('npm install', {cwd: project})
+
+  console.log(`Copying ${main} file...`)
```

Comment:
@HaidarJbeily7 let's move this log below after copying is done and print something like: `copied file to directory`